charts/prometheus-json-exporter/Chart.yaml

```diff
@@ -19,7 +19,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.2
+version: 0.1.3
```

Comment:
new feature should raise minor version

Reply:
I wasn't sure since the changes are disabled by default. I just bumped to `0.2.0`